apps/astarte_appengine_api/config/dev.exs

```diff
@@ -13,6 +13,8 @@ config :astarte_appengine_api, Astarte.AppEngine.APIWeb.Endpoint,
   check_origin: false,
   watchers: []
 
+config :astarte_appengine_api, :astarte_instance_id, ""
```

Comment:
This should not be needed as there's already Skogrsa handling the config and default (this holds for all other services, too)